apps/back-office/src/app/melding/[meldingId]/_components/AttachmentImage.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { Image } from '@amsterdam/design-system-react'
+import { render, screen } from '@testing-library/react'
+
+import { AttachmentImage } from './AttachmentImage'
+
+vi.mock('@amsterdam/design-system-react', async () => {
+  const actual = await vi.importActual('@amsterdam/design-system-react')
+  return {
+    ...actual,
+    Image: vi.fn(),
+  }
+})
+
+const createObjectURLMock = vi.fn().mockImplementation(() => {
+  return 'test-url'
+})
+
+global.URL.createObjectURL = createObjectURLMock
+
+describe('AttachmentImage', () => {
+  it('renders an image when a blob is provided', async () => {
+    render(<AttachmentImage blob={new Blob(['test-blob'], { type: 'image/jpeg' })} fileName={'IMG_0815.jpg'} />)
+
+    expect(createObjectURLMock).toHaveBeenCalled()
+
+    expect(Image).toHaveBeenCalledWith(
+      expect.objectContaining({
+        src: 'test-url',
+        alt: '',
+      }),
+      undefined,
+    )
```

Comment:
We can do this here, and remove the mock:

```suggestion
    const image = screen.getByRole('presentation')

    expect(image).toHaveAttribute('src', 'test-url')
```